packages/gatsby/src/schema/resolvers.js

```diff
@@ -74,7 +74,7 @@ const group = (source, args, context, info) => {
         acc[key] = (acc[key] || []).concat(node)
       })
     return acc
-  }, {})
+  }, Object.create(null))
```

Comment:
do you think this warrants a comment like `// Fixes #[issue]` ? I could see myself stumbling across this code and changing it to a `{}` without thinking too hard.

Reply:
I agree with @blainekasten, a comment that an object without prototype fixes it.

Reply:
It makes perfect sense for me. Added a comment!

Reply:
On the other hand, changing this would fail the test %)